### PR TITLE
Identify: doc on polygon and polyline usage

### PR DESCRIPTION
**Pull request by loicgasser**
This PR adds:
* some doc on how to use polygons and polylines with identify
* some tests for polygons and polylines with identify

### CODE REVIEW

File: chsdi/static/doc/source/services/sdiservices.rst [hunk 1]
```diff
@@ -265,7 +265,9 @@ No more than 50 features can be retrieved per request.
 | Parameters                        | Description                                                                               |
 +===================================+===========================================================================================+
 | **geometry (required)**           | The geometry to identify on. The geometry is specified by the geometry type.              |
-|                                   | This parameter is specified as a separated list of coordinates.                           |
+|                                   | This parameter is specified as a separated list of coordinates. The simple syntax (comma  |
+|                                   | separated list of coordinates) and the complex one can be used. (`ESRI syntax for         |
+|                                   | geometries <http://help.arcgis.com/en/arcgisserver/10.0/apis/rest/geometry.html>`_)       |
```
Comment (procrastinatio) [inline chsdi/static/doc/source/services/sdiservices.rst]: ESRI has radically changed its wesite, maybe use this  link http://resources.arcgis.com/en/help/arcgis-rest-api/index.html#//02r3000000n1000000

**Comment by procrastinatio** [comment]
OK, tests and examples work.

File: chsdi/static/doc/source/services/sdiservices.rst (hunk 1 above)
Comment (loicgasser) [inline chsdi/static/doc/source/services/sdiservices.rst]: quite nicer indeed.

Comment (loicgasser) [inline chsdi/static/doc/source/services/sdiservices.rst]: I'm changing the link

**Comment by loicgasser** [comment]
Adapted accroding to your comments thanks

**Pull request merged**